packages/core/docs/recipes/redis.md

```diff
@@ -0,0 +1,101 @@
+# Use of redis{Cluster|Sentinel} plugins
+Please read [redisCluster](../reference/redis/cluster.md) and [redisSentinel](../reference/redis/sentinel.md) plugins reference first.
+
+## Recipies:
+* [Redis Cluster](#connect-to-redis-cluster)
+* [Redis Sentinel](#connect-to-redis-sentinel)
+* [HTTP action with Redis command](#create-action-using-redis-family-plugins)
+* [Perform additional operations on Successfull connection](#perform-additional-operations)
+## Connect to Redis Cluster
+### 1. Enable plugin
+Enable `RedisCluster` plugin in your service configuration by adding it into `config.plugins` list:
+
+```js
+// configs/plugins.js
+exports.plugins = [
+  'validator',
+  'redisCluster'
+];
+```
+
+### 2. Plugin configuration
+Connect to Redis Cluster with nodes running on the same server but with different port:
+```js
+// configs/redis.js
+exports.redis = {
+  options: {
+    keyPrefix: '{myPrefix}',
+    dropBufferSupport: false,
+  },
+  luaScripts: '/path/to/my/cool-lua-scripts/',
+  hosts: Array.from({ length: 3 }).map((_, i) => ({
+    host: '172.16.0.10',
+    port: 7000 + i,
+  })),
+}
+```
+
+## Connect to Redis Sentinel
+### 1. Enable plugin
+Enable `redisSentinel` plugin in your service configuration by adding it into `config.plugins` list:
+
+```js
+// configs/plugins.js
+exports.plugins = [
+  'validator',
+  'redisSentinel'
+];
+```
+
+### 2. Plugin configuration
+Connect to Redis Sentinel running on local server:
+```js
+// configs/redis.js
+exports.redis = {
+  options: {
+    keyPrefix: '{myPrefix}',
+    dropBufferSupport: false,
+  },
+  luaScripts: '/path/to/my/cool-lua-scripts/',
+  name: 'redis-group-name'
+  sentinels: [{
+    { host: "localhost", port: 26379 },
+  }],
+}
+```
+
+## Create Action using Redis Family plugins
+Get Members from SET `my-key` and return them on HTTP Request:
+```js
+// actions/redisAction.js
```

Comment:
src/actions/redisAction.js
Current recommended structure is documented here: https://github.com/microfleet/core/blob/master/packages/core/docs/recipes/structure.md